kernel/tests/log_compaction.rs

```diff
@@ -0,0 +1,209 @@
+use std::sync::Arc;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use delta_kernel::engine::to_json_bytes;
+use delta_kernel::schema::{DataType, StructField, StructType};
+use delta_kernel::Snapshot;
+use object_store::path::Path;
+use object_store::ObjectStore;
+use test_utils::{create_table, engine_store_setup};
+use url::Url;
```

Comment:
nit on breaking up imports
```suggestion
use delta_kernel::engine::to_json_bytes;
use delta_kernel::schema::{DataType, StructField, StructType};
use delta_kernel::Snapshot;
use test_utils::{create_table, engine_store_setup};

use object_store::path::Path;
use object_store::ObjectStore;
use url::Url;
```

Reply:
(I realize this isn't codified anywhere - i'll add something to `CONTRIBUTING.md`)

Reply:
Done.